src/module/apps/effects-panel.ts

```diff
@@ -21,18 +20,21 @@ export class EffectsPanel extends appv1.api.Application {
      */
     refresh = fu.debounce(this.render.bind(this), 100);
 
-    static override get defaultOptions(): appv1.api.ApplicationV1Options {
-        return {
-            ...super.defaultOptions,
-            id: "pf2e-effects-panel",
-            popOut: false,
+    static override DEFAULT_OPTIONS: DeepPartial<fa.ApplicationConfiguration> = {
+        id: "pf2e-effects-panel",
```

Comment:
```suggestion
        id: "effects-panel",
```
Set `root: true` and unwrap the template